openbook/settings.py

```diff
@@ -456,6 +459,9 @@
 MODERATED_OBJECT_DESCRIPTION_MAX_LENGTH = 1000
 GLOBAL_HIDE_CONTENT_AFTER_REPORTS_AMOUNT = int(os.environ.get('GLOBAL_HIDE_CONTENT_AFTER_REPORTS_AMOUNT', '20'))
 MODERATORS_COMMUNITY_NAME = os.environ.get('MODERATORS_COMMUNITY_NAME', 'mods')
+POST_LINK_WHITELIST_DOMAIN_CACHE_TIMEOUT = 60*60*24  # one day
```

Comment:
Can we make this an environment variable based on hours?

`60*60*int(os.environ.get('POST_LINK_WHITELIST_DOMAIN_CACHE_TIMEOUT', '24'))`